client/src/pages/Profile/Profile.tsx

```diff
@@ -91,13 +95,43 @@ export default function Profile(): JSX.Element {
     );
   };
 
+  const submitNewPic = async (e: ChangeEvent<HTMLInputElement>) => {
+    if (e.target.files) {
+      setSelectedFile(e.target.files[0]);
+      try {
+        const formData = new FormData();
+        formData.append('image', e.target.files[0], e.target.files[0].name);
+        const newPic = await updateProfile(formData);
+        setNewProfilePic(newPic);
+      } catch (err) {
+        console.log(err);
```

Comment:
Can we update the snack bar here rather than logging the error to console?

Reply:
This is done